pennylane/templates/subroutines/approx_time_evolution.py

```diff
@@ -24,6 +24,12 @@ class ApproxTimeEvolution(Operation):
     r"""Applies the Trotterized time-evolution operator for an arbitrary Hamiltonian, expressed in terms
     of Pauli gates.
 
+        .. note::
```

Comment:
Does it make any sense to mention this new feature in `qml.evolve` docs?

Reply:
Nope, it should be independent of it.